demo.py

```diff
@@ -42,9 +42,11 @@ def run_demo():
     # ipsec stats
     print(primary_asa.ipsec_stats())
 
-    # context details
-    print(primary_asa.show_context_details())
```

Comment:
Why remove this? This was part of another teams sprint and should remain part of the codebase. Please add it back in and we can approve this pull request

Reply:
Oh, I never deleted that. I am not sure how that happened. I just added my stub function. @dmeyer3691

Reply:
I added back the show context details stub function @dmeyer3691

Reply:
Thanks, looks good now!